<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Bunny Image Analizer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="iconos/Iconos.qrc">
    <normaloff>:/iconos/conejito.jpg</normaloff>:/iconos/conejito.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="imagenOriginal">
      <property name="minimumSize">
       <size>
        <width>532</width>
        <height>546</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>532</width>
        <height>546</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="imagenEditada">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
   </widget>
   <addaction name="menuArchivo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="actionAbrir"/>
   <addaction name="actionGuardar"/>
   <addaction name="actionCerrar"/>
   <addaction name="separator"/>
   <addaction name="actionDeshacer"/>
   <addaction name="actionRehacer"/>
   <addaction name="actionFijar"/>
   <addaction name="separator"/>
   <addaction name="actionUmbralizar"/>
   <addaction name="actionEscalaDeGrises"/>
   <addaction name="actionRGB"/>
  </widget>
  <action name="actionAbrir">
   <property name="icon">
    <iconset resource="iconos/Iconos.qrc">
     <normaloff>:/iconos/folder_add.png</normaloff>:/iconos/folder_add.png</iconset>
   </property>
   <property name="text">
    <string>Abrir</string>
   </property>
   <property name="toolTip">
    <string>Abrir archivo</string>
   </property>
   <property name="statusTip">
    <string>Abriendo archivo</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="icon">
    <iconset resource="iconos/Iconos.qrc">
     <normaloff>:/iconos/Actions-document-save-icon.png</normaloff>:/iconos/Actions-document-save-icon.png</iconset>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
   <property name="toolTip">
    <string>Guardar imagen</string>
   </property>
  </action>
  <action name="actionUmbralizar">
   <property name="icon">
    <iconset>
     <normaloff>iconos/umbral.png</normaloff>iconos/umbral.png</iconset>
   </property>
   <property name="text">
    <string>Umbralizar</string>
   </property>
   <property name="toolTip">
    <string>Umbralizar imagen</string>
   </property>
  </action>
  <action name="actionCerrar">
   <property name="icon">
    <iconset resource="iconos/Iconos.qrc">
     <normaloff>:/iconos/cerrar.ico</normaloff>:/iconos/cerrar.ico</iconset>
   </property>
   <property name="text">
    <string>Cerrar </string>
   </property>
   <property name="toolTip">
    <string>Cerrar imagen</string>
   </property>
  </action>
  <action name="actionFijar">
   <property name="icon">
    <iconset resource="iconos/Iconos.qrc">
     <normaloff>:/iconos/palomita.png</normaloff>:/iconos/palomita.png</iconset>
   </property>
   <property name="text">
    <string>fijar</string>
   </property>
   <property name="toolTip">
    <string>Fijar imagen editada</string>
   </property>
  </action>
  <action name="actionDeshacer">
   <property name="icon">
    <iconset resource="iconos/Iconos.qrc">
     <normaloff>:/iconos/deshacer-icono-5403-128.png</normaloff>:/iconos/deshacer-icono-5403-128.png</iconset>
   </property>
   <property name="text">
    <string>deshacer</string>
   </property>
   <property name="toolTip">
    <string>Deshacer</string>
   </property>
  </action>
  <action name="actionEscalaDeGrises">
   <property name="icon">
    <iconset>
     <normaloff>iconos/images.jpeg</normaloff>iconos/images.jpeg</iconset>
   </property>
   <property name="text">
    <string>escalaDeGrises</string>
   </property>
   <property name="toolTip">
    <string>Convierte la imagen a escala de grises</string>
   </property>
  </action>
  <action name="actionRGB">
   <property name="icon">
    <iconset resource="iconos/Iconos.qrc">
     <normaloff>:/iconos/rgb.png</normaloff>:/iconos/rgb.png</iconset>
   </property>
   <property name="text">
    <string>RGB</string>
   </property>
   <property name="toolTip">
    <string>Escoger un canal RGB</string>
   </property>
  </action>
  <action name="actionRehacer">
   <property name="icon">
    <iconset resource="iconos/Iconos.qrc">
     <normaloff>:/iconos/rehacer-icono-5403-128.png</normaloff>:/iconos/rehacer-icono-5403-128.png</iconset>
   </property>
   <property name="text">
    <string>rehacer</string>
   </property>
   <property name="toolTip">
    <string>Rehacer</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="iconos/Iconos.qrc"/>
 </resources>
 <connections/>
</ui>
